applications/app/templates/webAppManifest.scala.js

```diff
@@ -1,34 +0,0 @@
-@import conf.Static
```

Comment:
I feel like the manifest file is still useful as it allows us to provide a reasonable experience to users who happen to 'install' the website. We actually had feedback from a user who did just that recently so I would say it's not unreasonable to maintain this.